### ISSUE TITLE
Hide editable field BG not working in conjunction with Read Fields from PDF

### ISSUE THREAD

**Issue by CePeU** [farling42/fvtt-pdf-pager#38]
Using both parameters "Hide editable field BG = enabled" and "Read Fields from PDF = enabled" will still show the editable field background. A switch would of course be possible once the PDF fields have been read and overwritten but the expectation would be that "Hide editable field BG" would also work in conjunction with "Read Fields from PDF = enabled".

**Comment by farling42** [comment]
Do you play with the "read fields from PDF" always enabled?

I envisaged that setting would only be on temporarily while the PDF was being loaded into Foundry.

I ask because when the setting is enabled, then no updates made to the Actor within Foundry are copied into the displayed PDF.

**Comment by farling42** [comment]
Support added in 0.34.0
5aa04b8a203a83093b2a23f7118c5ed1c9334e8f

**Comment by CePeU** [comment]
Actually I kind of stumbled uppon this as at first my custom PDF did not show anything but I was sure that it worked. I realized that something is amiss and finally figured out that obviously I overwrite the fields on first loading an new PDF. So I set all things up and was surprised that BG was on ... In the end we probably will not play with "read fields form PDF" always enabled. On the other hand we might do a lot of mixing and maybe PDF might be kept and updated by players outside of foundry. Long term I envision to be able to migrate to self coded foundry character sheets (if I ever find time ...). For now PDF are perfect and your work shows steady progress and improvement! I am really quite enamored by your work.
So to finalize my rant - you are probably right we will not work with "read fields from PDF" allways on but it struck my inner monk that BG was on while I disabled it ;). Thanks again for your relentless and fast work!
